core/src/main/scala/com/salesforce/op/stages/impl/tuning/DataCutter.scala

```diff
@@ -86,16 +89,37 @@ class DataCutter(uid: String = UID[DataCutter]) extends Splitter(uid = uid) with
 
     import data.sparkSession.implicits._
 
-    val labels = data.map(r => r.getDouble(0) -> 1L)
-    val labelCounts = labels.groupBy(labels.columns(0)).sum(labels.columns(1)).persist()
-    val (resKeep, resDrop) = estimate(labelCounts)
+    val labels = data.map(r => r.getDouble(0))
+    val labelCounts = labels.groupBy(labels.columns(0)).count().persist()
```

Comment:
`.count().persist()` seems weird to me. what's the return type of `count()`?

Reply:
accidental leftover, did not remove enough stuff after getting rid of an unnecessary transformation `r.getDouble(0) -> 1L)`

Reply:
UPDATE: sorry, I got confused it belongs here for the same reason as before since we produce a df with `(label, count)` per [scaladoc](https://spark.apache.org/docs/2.2.0/api/scala/index.html#org.apache.spark.sql.RelationalGroupedDataset@count():org.apache.spark.sql.DataFrame)